lib/msal-common/src/telemetry/performance/PerformanceClient.ts

```diff
@@ -604,7 +610,8 @@ export abstract class PerformanceClient implements IPerformanceClient {
         for (const [id, cb] of this.callbacks) {
             if (cb.toString() === callback.toString()) {
                 this.logger.warning(
-                    `PerformanceClient: Performance callback is already registered with id: ${id}`
+                    `PerformanceClient: Performance callback is already registered with id: ${id}`,
+                    this.generateId()
```

Comment:
Why can't we use "" in this case?

Reply:
Good catch, thanks. I forgot to remove this. Fixed